routes/routes.py

```diff
@@ -18,25 +18,41 @@ def __init__(self, db, executor, caching_manager):
 
     def register_routes(self):
         @api_bp.route('/train')
-        def train(): return self.training_service.train()
+        def train():
+            logging.info("The route '/train' has been called.")
+            return self.training_service.train()
```

Comment:
can you think of a way to generate this call so the logging logic for this file will serve all the API call?

how can decorators help here? 
@Hazannovich  maybe you will be curious about this thing as well